src/traversal/printRelayOSSQuery.js

```diff
@@ -121,10 +121,8 @@ function printRoot(
     oneIndent + fieldName + children + newLine + '}';
 }
 
-function printMutation(
-  node: RelayQuery.Mutation,
-  printerState: PrinterState
-): string {
+function printOperation(node: RelayQuery.Operation, printerState: PrinterState): string {
+  const operationKind = node.getConcreteQueryNode().kind.toLowerCase();
```

Comment:
Yeah that might work, but then I would also want to change `node: RelayQuery.Operation` to `node: RelayQuery.Mutation | RelayQuery.Subscription` because I don't think it make any sense to send anything else into it since we only support Subscription and Mutation.

what do you think about that?